documentation/manual/src/main/asciidoc/en-US/modules/neo4j.asciidoc

```
@@ -752,8 +752,13 @@ The following summary will help you to keep track of the labels assigned to a ne
 [[ogm-neo4j-transactions]]
 === Transactions
 
-Neo4j operations can be executed only inside a transaction.
-Unless a different +org.hibernate.engine.transaction.jta.platform.spi.JtaPlatform+ is specified, OGM will use a specific implementation to integrate with the Neo4j transaction mechanism.
+[CAUTION]
+====
+Neo4J operations must be executed inside a transaction.
+Make sure your interactions with Hibernate OGM are within a transaction when you target Neo4J.
```

What happens if no TX is running? Do we silently fail or raise a meaningful exception?